scripts/generate_data_network.py

```diff
@@ -56,33 +71,82 @@ def generate_network(num_units, num_nights):
                                 pic, coords, start_time, end_time)
                 write_lightcurve("{}/{}".format(temp_dir, lc_filename), hdu)
 
-                pan_storage.upload("{}/{}".format(temp_dir, psc_filename), remote_path=psc_filename)
-                pan_storage.upload("{}/{}".format(temp_dir, lc_filename), remote_path=lc_filename)
+                pan_storage.upload(
+                    "{}/{}".format(temp_dir, psc_filename), remote_path=psc_filename)
+                pan_storage.upload(
+                    "{}/{}".format(temp_dir, lc_filename), remote_path=lc_filename)
+            print('{} observed {} stars on {}.'.format(
+                unit, stars_per_night, curr_date.strftime('%Y-%m-%d')))
+        curr_date = curr_date + timedelta(days=1)
 
 
-def set_cameras(cam_dict, unitid):
+def init_units(num_units):
+    """Initialize a network of new units and their cameras, assigning unique IDs."""
+    units = []
+    for i in range(num_units):
+        unit = "PAN{:03d}".format(i)
+        init_cameras(unit)
+        units.append(unit)
+    return units
+
+
+def get_current_network():
```

Comment:
with current setup, can't get data about the unit's site from the data already on cloud - if expand, should store metadata with unit somehow?